src/core/ext/filters/client_channel/lb_policy.h

```diff
@@ -261,8 +260,7 @@ class LoadBalancingPolicy : public InternallyRefCounted<LoadBalancingPolicy> {
   /// Note that the LB policy gets the set of addresses from the
   /// GRPC_ARG_SERVER_ADDRESS_LIST channel arg.
   virtual void UpdateLocked(const grpc_channel_args& args,
-                            RefCountedPtr<Config> lb_config) {
-    std::move(lb_config);  // Suppress clang-tidy complaint.
+                            RefCountedPtr<Config> lb_config) {  // NOLINT
```

Comment:
No need for the braces around `GRPC_ABSTRACT` here.

Reply:
Done!